modules/json/JsonMngr.h

```diff
@@ -0,0 +1,196 @@
+// vim: set ts=4 sw=4 tw=99 noet:
+//
+// AMX Mod X, based on AMX Mod by Aleksander Naszko ("OLO").
+// Copyright (C) The AMX Mod X Development Team.
+//
+// This software is licensed under the GNU General Public License, version 3 or higher.
+// Additional exceptions apply. For full license details, see LICENSE.txt or visit:
+//     https://alliedmods.net/amxmodx-license
+
+//
+// JSON Class
+//
+
+#include <amxxmodule.h>
+#include <parson.h>
+#include <amtl/am-vector.h>
+#include <amtl/am-autoptr.h>
+#include <amtl/am-uniqueptr.h>
+#include <amtl/am-deque.h>
+
+#include "IJsonMngr.h"
+
+using namespace AMXX;
+
+class JSONMngr : public IJSONMngr
+{
+	public:
+
+	JSONMngr() = default;
+	virtual ~JSONMngr();
+
+	// Handles
+	virtual bool IsValidHandle(JS_Handle id, JSONHandleType type = Handle_Value);
```

Comment:
You should use `override` here and with any other overridden method instead of `virtual`. So instead of
```Cpp
virtual bool IsValidHandle(JS_Handle id, JSONHandleType type = Handle_Value);
```
it should be
```Cpp
bool IsValidHandle(JS_Handle id, JSONHandleType type = Handle_Value) override;
```

Here `virtual` indicates that given method may be overridden further by a derived class. `override` ensures that the overridden method exists in the base class (`IJSONMngr`) and is of the same type.

Reply:
Done.